nix/shell.nix

```diff
@@ -408,6 +408,38 @@
               $REWARDS
           '';
         }
+        {
+          category = "multisig";
+          name = "process-queue-rewards";
+          help = "Construct batch tx to process withdraw queue with rewards and send tx to safe for other signatures";
+          command = ''
+            forge script $PRJ_ROOT/script/ProcessQueueRewards.s.sol:ProcessQueueRewardsScript \
+              --chain-id 1 \
+              --rpc-url $RPC_MAINNET \
+              --broadcast \
+              --private-key $PRIVATE_KEY \
+              --verify \
+              --etherscan-api-key $ETHERSCAN_API_KEY \
+              --ffi \
+              -vvvvv
+          '';
+        }
+        {
+          category = "multisig";
+          name = "process-queue-exits";
+          help = "Construct batch tx to process withdraw queue with exits and send tx to safe for other signatures";
+          command = ''
+            forge script $PRJ_ROOT/script/ProcessQueueExits.s.sol:ProcessQueueExitsScript \
+              --chain-id 1 \
+              --rpc-url $RPC_MAINNET \
+              --broadcast \
+              --private-key $PRIVATE_KEY \
+              --verify \
+              --etherscan-api-key $ETHERSCAN_API_KEY \
```

Comment:
```suggestion
              --private-key $PRIVATE_KEY \
              --skip-simulation \
```

Reply:
remove broadcast and verify options since they are not required for constructing batch to gnosis safe

Reply:
https://github.com/manifoldfinance/mevETH2/pull/273/commits/d77754061765961c12ebf04f9abdcb909c4c8983